integration/upstream_client_test.go

```diff
@@ -205,11 +206,12 @@ func setup(
 	srv3 := gcpserverv3.NewServer(ctx, nil, nil)
 
 	// Start the origin server
-	go gcptest.RunManagementServer(ctx, srv2, srv3, originServerPort)
+	go gcptest.RunHTTP(ctx, originServerHTTPPort)
```

Comment:
What's the distinction between these two servers / why does the origin server need both ports?